src/index.js

```diff
@@ -0,0 +1,171 @@
+import './style.css';
+
+const list = JSON.parse(localStorage.getItem('list')) || [{
+  descreption: 'first',
+  completed: false,
+},
+{
+  descreption: 'second',
+  completed: false,
+},
+{
+  descreption: 'third',
+  completed: false,
+}];
+const insert = document.getElementById('insert');
+const listItems = document.querySelector('.list-items');
+function ToDo(descreption, completed, index) {
+  this.descreption = descreption;
+  this.completed = completed;
+  this.index = index;
+}
+
+function removeChildNodes(listItems) {
+  while (listItems.firstChild) {
+    listItems.removeChild(listItems.firstChild);
+  }
+}
+
+function refreshStorage() {
+  localStorage.setItem('list', JSON.stringify(list));
+}
+
+function update() {
+  for (let i = 0; i < list.length; i += 1) {
+    list[i].index = i;
+    refreshStorage();
+  }
+}
+
+function clearAll() {
+  const clearAllBtn = document.querySelector('.remove-button');
+  if (listItems.childNodes.length > 0) {
+    clearAllBtn.style.display = 'block';
+  } else {
+    clearAllBtn.style.display = 'none';
+  }
+  clearAllBtn.addEventListener('click', () => {
+    for (let i = 0; i < list.length; i += 1) {
+      if (list[i].completed) {
+        list.splice(i, 1);
+        listItems.removeChild(listItems.childNodes[i]);
+        window.location.reload();
+        refreshStorage();
+        update();
+      }
+    }
+  });
+}
```

Comment:
- [ ] This method is also buggy. When Task1 is once removed and later while I try to clear all completed to-do, it keeps the top to-do undelete. Kindly fix this bug. :pray: 

When about to clear all completed
![Screenshot from 2021-10-29 22-27-56](https://user-images.githubusercontent.com/64894927/139491726-51336d40-f49b-4d19-9f19-000179be1b45.png)

After hitting the `clear all completed` button
![Screenshot from 2021-10-29 22-28-01](https://user-images.githubusercontent.com/64894927/139491753-2b56e09e-46e1-4cbd-892d-b06518cc1155.png)